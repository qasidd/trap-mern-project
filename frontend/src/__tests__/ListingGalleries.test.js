import ListingGalleries from '../components/Galleries/ListingsGallery';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<ListingGalleries/>).toJSON();
    expect(instance).toMatchSnapshot();
    
})