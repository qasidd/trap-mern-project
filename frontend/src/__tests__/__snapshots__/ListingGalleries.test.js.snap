// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
<div
  className="mainContent container"
>
  <h1
    className="display-4"
  >
    Now Showing
  </h1>
  <div
    className="row mt-4"
  >
    <div
      className="col-12 text-center"
    >
      <p>
        Loading...
      </p>
      <div
        animation="border"
        className="spinner-border"
        role="status"
      >
        <span
          className="sr-only"
        >
          Loading...
        </span>
      </div>
    </div>
  </div>
</div>
`;
